pkg/driver/camera/camera_linux.go

```diff
@@ -23,6 +24,7 @@ import (
 const (
 	maxEmptyFrameCount = 5
 	prioritizedDevice  = "video0"
+	bufferedFrameCount = 1
```

Comment:
instead of making this a const, we know that we don't really ever want to be buffering more than this, so I think we can keep the 1 below

Reply:
There's a comment above `SetBufferCount` that suggest it may be eliminated in the future so it may not always be 1 and might go away. If that happens then the context will be lost for why we're flushing a single frame. Either way, I don't have a strong opinion on this so removed.